src/controllers/emailController.spec.js

```diff
@@ -0,0 +1,148 @@
+const { mockReq, mockRes, assertResMock } = require('../test');
+const emailController = require('./emailController');
+const jwt = require('jsonwebtoken');
+const EmailSubcriptionList = require('../models/emailSubcriptionList');
+const userProfile = require('../models/userProfile');
+// const emailSender = require('../utilities/emailSender');
+
+jest.mock('jsonwebtoken');
+jest.mock('../models/userProfile');
+jest.mock('../models/emailSubcriptionList');
+jest.mock('../utilities/emailSender');
+
+
+
+
+const makeSut = () => {
+  const {
+    sendEmail,
+    sendEmailToAll,
+    updateEmailSubscriptions,
+    addNonHgnEmailSubscription,
+    removeNonHgnEmailSubscription,
+    confirmNonHgnEmailSubscription,
+  } = emailController;
+  return {
+    sendEmail,
+    sendEmailToAll,
+    updateEmailSubscriptions,
+    addNonHgnEmailSubscription,
+    removeNonHgnEmailSubscription,
+    confirmNonHgnEmailSubscription,
+  };
+};
+describe('emailController Controller Unit tests', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('sendEmail function', () => {
+  test('should send email successfully', async () => {
+    const { sendEmail } = makeSut();
+    const mockReq = {
```

Comment:
This is the right idea, but you need to be careful not to reset the entire `mockReq` object. Instead you want to just append on the fields you need:
`mockReq.body.to = "asdf@asdf.com"`

This is ok for the unit tests, but if you do this for the integration tests things will start to break.